@white-axe do you have any ideas on why a subset of the builds are failing with the new FluidSynth? E.g. Trixie/POWER8 shows this:

```
/__w/mkxp-z/mkxp-z/linux/downloads/power8le/fluidsynth/src/gentables/fluid_concave.cpp:17:33: error: '(-4.0e+2l / 9.6e+2l)' is not a constant expression
```

--- a/linux/Makefile
+++ b/linux/Makefile
@@ -300,7 +300,7 @@ $(DOWNLOADS)/fluidsynth/cmakebuild/Makefile: $(DOWNLOADS)/fluidsynth/CMakeLists.
 
 $(DOWNLOADS)/fluidsynth/CMakeLists.txt:
 	$(CLONE) $(GITHUB)/FluidSynth/fluidsynth --recurse-submodules $(DOWNLOADS)/fluidsynth; \
-	cd $(DOWNLOADS)/fluidsynth; git checkout c5125b5dfee828ea715618d620651b6d135a39cb
+	cd $(DOWNLOADS)/fluidsynth; git checkout a8c8a4f28c617ddda0519d51350901b75d75882e
 
 # OpenSSL
 openssl: init_dirs $(LIBDIR)/libssl.a

--- a/windows/Makefile
+++ b/windows/Makefile
@@ -280,7 +280,7 @@ $(DOWNLOADS)/fluidsynth/cmakebuild/Makefile: $(DOWNLOADS)/fluidsynth/CMakeLists.
 
 $(DOWNLOADS)/fluidsynth/CMakeLists.txt:
 	$(CLONE) $(GITHUB)/FluidSynth/fluidsynth --recurse-submodules $(DOWNLOADS)/fluidsynth; \
-	cd $(DOWNLOADS)/fluidsynth; git checkout c5125b5dfee828ea715618d620651b6d135a39cb
+	cd $(DOWNLOADS)/fluidsynth; git checkout a8c8a4f28c617ddda0519d51350901b75d75882e
 
 # OpenSSL
 openssl: init_dirs $(LIBDIR)/libssl.a

